public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -851,22 +851,20 @@ export class CloudWatchDatasource
         return { ...result, [key]: null };
       }
 
-      const newValues = this.getVariableValue(value, scopedVars);
+      const newValues = this.expandVariableToArray(value, scopedVars);
       return { ...result, [key]: newValues };
     }, {});
   }
 
   // get the value for a given template variable
```

Comment:
Is this comment still valid now with this change?

Reply:
I think so, it still gets the value